cni/go.mod

```diff
@@ -0,0 +1,83 @@
+module github.com/Azure/azure-container-networking/cni
+
+go 1.17
+
+replace (
+	github.com/Azure/azure-container-networking => ../
+	github.com/Azure/azure-container-networking/cns => ../cns
+	github.com/Microsoft/go-winio => github.com/microsoft/go-winio v0.4.17
+	github.com/Microsoft/hcsshim => github.com/microsoft/hcsshim v0.8.22
+	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.12.0
+	github.com/onsi/gomega => github.com/onsi/gomega v1.10.0
+)
+
+require (
+	github.com/Azure/azure-container-networking v0.0.0-00010101000000-000000000000
```

Comment:
what this v0.0.0-00010101000000-000000000000 signifies?

Reply:
go mod uses the 0.0.0 version when we do `replace` to a filesystem path, since it is not fetching a published module version